frontend/packages/dev-console/src/components/import/icon/IconDropdown.scss

```diff
@@ -0,0 +1,20 @@
+.icon-dropdown {
+  img {
+    margin-right: var(--pf-global--spacer--sm);
+  }
+
+  .pf-c-dropdown__toggle-text {
+    display: flex;
+    align-items: center;
+  }
+
+  &__menu {
+    overflow: overlay;
```

Comment:
```suggestion
```
Not officially supported.

Reply:
I just removed it now, the max-height is enough here to ensure that the scrollable is not going outside the screen.

Reply:
Confused by your comment. 

I was just pointing out [MDN says it's not officially supported](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow#Values):
> overlay 
> Behaves the same as auto, but with the scrollbars drawn on top of content instead of taking up space. Only supported in WebKit-based (e.g., Safari) and Blink-based (e.g., Chrome or Opera) browsers.

Reply:
I removed the overflow: overlay because it is not required.

I kept the max-height.